### PR TITLE
[Hub Generated] Review request for Microsoft.Consumption to add version stable/2019-10-01

### PR DESCRIPTION
**Pull request by AlahmadiQ8**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/AlahmadiQ8/azure-rest-api-specs/dev-consumption-Microsoft.Consumption-2019-10-01-m...master/).

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [x] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by AlahmadiQ8** [comment]
@weidongxu-microsoft This PR fixes the S360 errors for the Usage Details List endpoint owned by the Azure Cost Management team. 

https://msazure.visualstudio.com/Swagger%20KPIs/_workitems/edit/8187825
https://vnext.s360.msftcloudes.com/blades/unifiedplatform?blade=KPI:cf751b97-2c95-420f-baa8-9a16e8643a5b~SLA:2~AssignedTo:All~Forums:All~_loc:UnifiedPlatform&def=&peopleBasedNodes=laagrawa_team;raphaelc_team;ashwinsa_team;mawadwan_team;adwise_team;pamog_team;shbharadwaj_team&tile=S360_ServiceId:80383977-026e-47cb-8a1b-7e7c3eff7350~_loc:__key__UnifiedPlatform__cf751b97-2c95-420f-baa8-9a16e8643a5b

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json [hunk 1]
```diff
@@ -152,6 +152,9 @@
               "$ref": "#/definitions/UsageDetailsListResult"
             }
           },
+          "503": {
```
Comment (weidongxu-microsoft) [inline specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json:155]: I don't think 503 need to be added separately. It should be same as `default` below.

If you add 503 here, it actually means 503 is now the expected return code, and client should be good with it and proceed.

**Comment by AlahmadiQ8** [comment]
@weidongxu-microsoft I undid the 503 separate response. We will fix it in the api such that it returns a default response. 

This pr now only fixes missing property and spelling typos

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
LGTM.

For adding the property, it is flagged as breaking change, you would need signoff from JeffreyRichter.

**Comment by AlahmadiQ8** [comment]
Thank you so much. Can I get sign off on this one @JeffreyRichter. I'm hoping it still falls within the [Window to Fix](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/405/Window-to-Fix-Broken)

**Review by JeffreyRichter — APPROVED** [review]

**Pull request merged**